src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -77,6 +79,18 @@ public class WebView: EvasObject
         /// <since_tizen> 4 </since_tizen>
         public event EventHandler<SmartCallbackArgs> UrlChanged;
 
+        /// <summary>
+        /// Event that occurs when the title of the main frame is changed.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public event EventHandler<ContextMenuArgs> ContextMenuCustomize;
+
+        /// <summary>
+        /// Event that occurs when the URL of the main frame is changed.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public event EventHandler<ContextMenuArgs> ContextMenuItemSelected;
```

Comment:
In this case, a type of EventArgs should be ContextMenuItemEventArgs.

Reply:
Done